packages/plugins/src/dashboard-components/layers/absolute-position-layout.js

```diff
@@ -10,9 +10,14 @@ export default class AbsolutePositioningLayout extends Layer {
   #interactive;
   #selectionBox = this.createSelectionBox();
 
+  snappingEnabled;
+  gridSize;
```

Comment:
I would rename these to `#snappingEnabled` and `#gridSize`. The `#` is the private modifier in javascript, a feature that was introduced in es2017 or something.